### DISCUSSION TITLE
React Testing Library dependency is not included in the `jaeger-ui-components`

### DISCUSSION THREAD

**Discussion by xrisaD** [grafana/grafana#49085] [Alerting]
The React Testing Library dependency is not included in the devDependencies in the `packages/jaeger-ui-components/package.json` so RTL tests cannot be written for the components.
It should be added to move forward with some of these issues: #48253

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
